lib/ZwaveClient.js

```diff
@@ -1425,7 +1425,7 @@ class ZwaveClient extends EventEmitter {
   /**
    * Send a command
    *
-   * @param {{nodeId: number, endpoint: number, commandClass: number}} ctx context to get the instance to send the command
+   * @param {{nodeId: number, endpoint: number, commandClass: number | string}} ctx context to get the instance to send the command
```

Comment:
The correct type for `commandClass` would be: `CommandClasses | keyof typeof CommandClasses`, where `CommandClasses` is the enum from `zwave-js`.